dc/core/PDFContent.cpp

```diff
@@ -47,6 +47,8 @@
 
 BOOST_CLASS_EXPORT_GUID( PDFContent, "PDFContent" )
 
+const qreal PDFContent::maxScale = 6;
```

Comment:
the 'other' content maxScale can be changed from the configuration file. It seems we need/have two different scales, one for vectorial content and one for rasterized content. Both should be configurable from the configuration imo.